scripts/variantstore/wdl/GvsValidateVAT.wdl

```diff
@@ -245,7 +258,7 @@ task SchemaOnlyOneRowPerNullTranscript {
             transcript_source is NULL AND
             transcript is NULL
         GROUP BY vid
-        HAVING num_rows = 1' > bq_variant_count.csv
+        HAVING num_rows > 1' > bq_variant_count.csv
```

Comment:
nice catch!

Reply:
nit: while you are here, there's a tiny typo below:
"# if the result of the query has any rows, that means there were vids will null transcripts and multiple"
 I think will should be with?